lib/quantity.ex

```diff
@@ -285,4 +323,62 @@ defmodule Quantity do
       "~Q[#{@for.to_string(quantity)}]"
     end
   end
+
+  # Normalizes unit to a standard form:
+  # * Shorten unit as much as possible
+  # * At most one :div (with possibly many :mults on each side)
+  # * All :mult units are sorted
+  # * Extra 1-units are removed
+  defp normalize_unit(unit) do
```

Comment:
:tada: Really nice!

Reply:
Niels-two-hours-ago is happy with this :smile:

Reply:
:slightly_smiling_face: :+1: